app/assets/javascripts/device_location.js

```diff
@@ -0,0 +1,43 @@
+function showAddress(lat, lng){
+	alert(lat + ' ' + lng)
+}
```

Comment:
It's generally OK in JS to use tabs for indentation. I mean it comes down to personal preference. There's a whole emotional conversation about it here. http://programmers.stackexchange.com/questions/57/tabs-versus-spaces-what-is-the-proper-indentation-character-for-everything-in-e

Since this is a rails project, I'll make the tabs 2 spaces for consistency.